Add version parameters to KCP CLI on Kyma upgrade (#554)
kyma-project/control-plane: docs/cli/commands/kcp_upgrade_kyma.md

Upgrades or reconfigures Kyma on targets of Runtimes.
The upgrade is performed by Kyma Control Plane (KCP) within a new orchestration asynchronously. The ID of the orchestration is returned by the command upon success.
The targets of Runtimes are specified via the `--target` and `--target-exclude` options. At least one `--target` must be specified.

> The version is specified via the `--version` (or -v) options. If empty, the version will be configured by Kyma Environment Broker (KEB).

Reviewer: ```suggestion
The version is specified using the `--version` (or -v) option. If not specified, the version is configured by Kyma Environment Broker (KEB).
```

  kcp upgrade kyma --target "account=CA.*"                       Upgrade Kyma on Runtimes of all global accounts starting with CA.
  kcp upgrade kyma --target all --target-exclude "account=CA.*"  Upgrade Kyma on Runtimes of all global accounts not starting with CA.
  kcp upgrade kyma --target "region=europe|eu|uk"                Upgrade Kyma on Runtimes whose region belongs to Europe.

>   kcp upgrade kyma --target all --version "master-00e83e99"      Upgrade Kyma on Runtimes of all global accounts to custom Kyma version master-00e83e99.

Reviewer: ```suggestion
  kcp upgrade kyma --target all --version "master-00e83e99"      Upgrade Kyma on Runtimes of all global accounts to the custom Kyma version (master-00e83e99).
```
